platforms/arm/nrf52/led_sysdefs_arm_nrf52.h

```diff
@@ -47,11 +47,12 @@ typedef __IO uint32_t RwReg;
 #define sei()  __enable_irq()
 
 #define FASTLED_NRF52_DEBUGPRINT(format, ...)
-//#define FASTLED_NRF52_DEBUGPRINT(format, ...)\
-//    do {\
-//        FastLED_NRF52_DebugPrint(format, ##__VA_ARGS__);\
-//    } while(0);
-
+/*
+#define FASTLED_NRF52_DEBUGPRINT(format, ...)\
+    do {\
+        FastLED_NRF52_DebugPrint(format, ##__VA_ARGS__);\
+    } while(0);
+*/
 
```

Comment:
this can be **safely** simplified, enabling/disabling with a single comment, from:
```c
#define FASTLED_NRF52_DEBUGPRINT(format, ...)
/*
#define FASTLED_NRF52_DEBUGPRINT(format, ...)\
    do {\
        FastLED_NRF52_DebugPrint(format, ##__VA_ARGS__);\
    } while(0);
*/
```
to:
```c
#define FASTLED_NRF52_DEBUGPRINT(format, ...) \
//    do { FastLED_NRF52_DebugPrint(format, ##__VA_ARGS__); } while(0)
```

Reply:
That might be a nice change.  
I'm going to merge the PR as-is, just to get the code out there today, but if you wanted to make an additional PR to streamline the #define.

Reply:
Thanks, @ardnew ... you're right, I like that design.

I'll have a PR ready soon ... see Issue #1023, which I just opened.  I'll submit a PR shortly.